python-test/features/agentProviderUi.feature

```diff
@@ -5,14 +5,14 @@ Feature: Create agents using orb ui
         Given the Orb user logs in through the UI
             And that fleet Management is clickable on ORB Menu
             And that Agents is clickable on ORB Menu
-        When a new agent is created through the UI
+        When a new agent is created through the UI with defined orb tag(s): region:br, demo:true, ns1:true
         Then the agents list and the agents view should display agent's status as New within 10 seconds
 
     Scenario: Provision agent
         Given the Orb user logs in through the UI
             And that fleet Management is clickable on ORB Menu
             And that Agents is clickable on ORB Menu
-        When a new agent is created through the UI
+        When a new agent is created through the UI with random orb tag(s): 2 tag(s)
```

Comment:
don't get the "2 tags()". what that does mean?

Reply:
it means the amount of tags that must be added to the agent or to the group..